keymanapp/keyman
# chore(common): Check in crowdin for Russian

Adds crowdin strings for Russian for locale ru-RU

## User Testing
This adds the crowdin strings for Russian locale ru-RU

TODO for @sgschantz 

## User Testing
Load the PR build for each platform and set the UI / Locale to "Pyccĸий (Russian)"

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> Pyccĸий (Russian)
3. Return to the Keyman app 
4. Verify the UI is in Russian (prompt, Settings strings, etc.).



* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the device locale to Russian
3. Return to the Keyman app
4. Verify the UI is in  Russian

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=ru-RU km-config
2. Verify Keyman UI is in Russian

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the locale to Russian
3. Verify the Keyman UI is in Russian

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to Pyccĸий (Russian)
3. Verify the Keyman UI is in Russian
